llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -954,6 +964,14 @@ def TuneSiFive7 : SubtargetFeature<"sifive7", "RISCVProcFamily", "SiFive7",
                                    [TuneNoDefaultUnroll,
                                     TuneShortForwardBranchOpt]>;
 
+def TuneVeyronFusions : SubtargetFeature<"ventana-veyron", "RISCVProcFamily", "VentanaVeyron",
```

Comment:
Please separate this out in a separate PR which only adds the proc family and the existing LUI/ADDI fusion.  Then this change can extend the list with the new fusions.

Reply:
done: https://github.com/llvm/llvm-project/pull/70414